apis/Google.Cloud.Diagnostics.AspNetCore/Google.Cloud.Diagnostics.AspNetCore/ErrorReporting/ErrorReportingExceptionLogger.cs

```diff
@@ -145,8 +145,8 @@ private HttpRequestContext CreateHttpRequestContext(HttpContext context)
             return new HttpRequestContext()
             {
                 Method = request?.Method?.ToString() ?? "",
-                Url = UriHelper.GetDisplayUrl(request) ?? "",
-                UserAgent = request?.Headers["User-Agent"] ?? "",
+                Url = request?.GetDisplayUrl() ?? "",
+                UserAgent = request?.Headers["User-Agent"].ToString() ?? "",
```

Comment:
So if these are StringValue values, do we need anything more than

```csharp
Url = request?.GetDisplayUrl(),
UserAgent = request?.Headers["User-Agent"]
```

Shouldn't we leave them assigning null if there's no value?